Sources/GDBRemote/Session.cpp

```diff
@@ -34,7 +34,7 @@ namespace ds2 {
 namespace GDBRemote {
 
 Session::Session(CompatibilityMode mode)
-    : _compatMode(mode), _threadsInStopReply(false) {
+    : SessionBase(mode), _threadsInStopReply(false) {
```

Comment:
I'm surprised we weren't calling the `SessionBase()` constructor explicitly before. This is good.